avl_tree/AvlTree.java

```diff
@@ -0,0 +1,306 @@
+import java.util.Scanner;
+import java.util.NoSuchElementException;
+import java.util.InputMismatchException;
+
+public class AvlTree {
+
+	public static void main(String[] args) {
+		AVL avl = new AVL();
+		Scanner inp = new Scanner(System.in);
+		while (true) {

```

Comment:
@Hsankesara I recommend you understand the concept of hardcoding inputs I'm damn sure you're taking it the wrong way. Please refer the main() of other implemented DS and compare you will get what hardcoding inputs is about.